presto-spi/src/main/java/com/facebook/presto/spi/block/RowBlock.java

```diff
@@ -37,20 +37,18 @@
     private final long retainedSizeInBytes;
 
     /**
-     * Create an row block directly from columnar nulls, field blocks, and offsets into the fields.
+     * Create an row block directly from columnar nulls, and field blocks.
      * A null row must have no entries.
```

Comment:
This javadoc line was probably referring to the check L48, now that check is removed do we still want to keep this line?